lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -0,0 +1,48 @@
+import classnames from 'classnames';
+import { useParams } from 'wouter-preact';
+
+import type { StudentStats } from '../../api-types';
+import { useConfig } from '../../config';
+import { apiCall } from '../../utils/api';
+import { useFetch } from '../../utils/fetch';
+import StudentsActivityTable from './StudentsActivityTable';
+
+export default function DashboardApp() {
+  const { assignmentId = 'unknown' } = useParams();
+  const { dashboard, api } = useConfig(['dashboard', 'api']);
+  const assignment = {
+    title: dashboard.assignment.title,
+    id: assignmentId,
+  };
+  const students = useFetch<StudentStats[]>(`${assignmentId}_students`, () =>
+    apiCall({
+      authToken: api.authToken,
+      method: 'GET',
+      path: `/api/assignment/${assignmentId}/stats`,
+    }),
+  );
+
+  return (
+    <div className="min-h-full bg-grey-2">
+      <div
+        className={classnames(
+          'flex justify-center p-3 mb-5 w-full',
+          'bg-white border-b shadow',
+        )}
+      >
+        <img
+          alt="Hypothesis logo"
+          src="/static/images/email_header.png"
+          className="h-10"
+        />
```

Comment:
A potential source of confusion to have an `email_header.png` file that is used from the web app.